{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Ha ocurrido un error inesperado.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"TypeError: Cannot read properties of undefined (reading 'hasOwnProperty')\n    at formatRepPar (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/queryConsulta.js:94:32)\n    at ExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/ProcSql.js:28:18)\n    at ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:22)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async ctrlExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Controler/contProcedure.js:16:29)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T16:45:12.136Z","ID_EVENTO":7310,"ID_PROCESO":6,"MSG_ERROR":"Error desconocido en Ejecucion de Procedimiento: {}.","level":"error","message":"Error ","timestamp":"2025-08-28 10:45:12"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Ha ocurrido un error inesperado.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"TypeError: Cannot read properties of undefined (reading 'hasOwnProperty')\n    at formatRepPar (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/queryConsulta.js:94:32)\n    at ExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/ProcSql.js:28:18)\n    at ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:22)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async ctrlExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Controler/contProcedure.js:16:29)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T16:45:13.190Z","ID_EVENTO":7311,"ID_PROCESO":6,"MSG_ERROR":"Error desconocido en Ejecucion de Procedimiento: {}.","level":"error","message":"Error ","timestamp":"2025-08-28 10:45:13"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Ha ocurrido un error inesperado.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"TypeError: Cannot read properties of undefined (reading 'hasOwnProperty')\n    at formatRepPar (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/queryConsulta.js:94:32)\n    at ExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/ProcSql.js:28:18)\n    at ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:22)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async ctrlExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Controler/contProcedure.js:16:29)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T16:45:15.227Z","ID_EVENTO":7312,"ID_PROCESO":6,"MSG_ERROR":"Error desconocido en Ejecucion de Procedimiento: {}.","level":"error","message":"Error ","timestamp":"2025-08-28 10:45:15"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Ha ocurrido un error inesperado.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"TypeError: Cannot read properties of undefined (reading 'hasOwnProperty')\n    at formatRepPar (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/queryConsulta.js:94:32)\n    at ExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/ProcSql.js:28:18)\n    at ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:22)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async ctrlExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Controler/contProcedure.js:16:29)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T16:46:39.531Z","ID_EVENTO":7313,"ID_PROCESO":6,"MSG_ERROR":"Error desconocido en Ejecucion de Procedimiento: {}.","level":"error","message":"Error ","timestamp":"2025-08-28 10:46:39"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Ha ocurrido un error inesperado.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"TypeError: Cannot read properties of undefined (reading 'hasOwnProperty')\n    at formatRepPar (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/queryConsulta.js:94:32)\n    at ExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/ProcSql.js:28:18)\n    at ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:22)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async ctrlExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Controler/contProcedure.js:16:29)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T16:51:37.718Z","ID_EVENTO":7314,"ID_PROCESO":6,"MSG_ERROR":"Error desconocido en Ejecucion de Procedimiento: {}.","level":"error","message":"Error ","timestamp":"2025-08-28 10:51:37"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Ha ocurrido un error inesperado.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"TypeError: Cannot read properties of undefined (reading 'hasOwnProperty')\n    at formatRepPar (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/queryConsulta.js:94:32)\n    at ExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/ProcSql.js:28:18)\n    at ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:22)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async ctrlExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Controler/contProcedure.js:16:29)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T16:51:38.781Z","ID_EVENTO":7315,"ID_PROCESO":6,"MSG_ERROR":"Error desconocido en Ejecucion de Procedimiento: {}.","level":"error","message":"Error ","timestamp":"2025-08-28 10:51:38"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Ha ocurrido un error inesperado.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"TypeError: Cannot read properties of undefined (reading 'hasOwnProperty')\n    at formatRepPar (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/queryConsulta.js:94:32)\n    at ExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/ProcSql.js:28:18)\n    at ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:22)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async ctrlExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Controler/contProcedure.js:17:29)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T16:53:01.076Z","ID_EVENTO":7316,"ID_PROCESO":6,"MSG_ERROR":"Error desconocido en Ejecucion de Procedimiento: {}.","level":"error","message":"Error ","timestamp":"2025-08-28 10:53:01"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Ha ocurrido un error inesperado.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"TypeError: Cannot read properties of undefined (reading 'hasOwnProperty')\n    at formatRepPar (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/queryConsulta.js:94:32)\n    at ExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/ProcSql.js:28:18)\n    at ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:22)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async ctrlExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Controler/contProcedure.js:17:29)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T16:53:02.121Z","ID_EVENTO":7317,"ID_PROCESO":6,"MSG_ERROR":"Error desconocido en Ejecucion de Procedimiento: {}.","level":"error","message":"Error ","timestamp":"2025-08-28 10:53:02"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Ha ocurrido un error inesperado.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"TypeError: Cannot read properties of undefined (reading 'hasOwnProperty')\n    at formatRepPar (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/queryConsulta.js:94:32)\n    at ExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/ProcSql.js:28:18)\n    at ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:22)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async ctrlExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Controler/contProcedure.js:17:29)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T16:54:04.280Z","ID_EVENTO":7318,"ID_PROCESO":6,"MSG_ERROR":"Error desconocido en Ejecucion de Procedimiento: {}.","level":"error","message":"Error ","timestamp":"2025-08-28 10:54:04"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Ha ocurrido un error inesperado.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"TypeError: Cannot read properties of undefined (reading 'hasOwnProperty')\n    at formatRepPar (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/queryConsulta.js:94:32)\n    at ExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/ProcSql.js:28:18)\n    at ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:22)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async ctrlExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Controler/contProcedure.js:17:29)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T16:54:39.468Z","ID_EVENTO":7319,"ID_PROCESO":6,"MSG_ERROR":"Error desconocido en Ejecucion de Procedimiento: {}.","level":"error","message":"Error ","timestamp":"2025-08-28 10:54:39"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Ha ocurrido un error inesperado.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"TypeError: Cannot read properties of undefined (reading 'hasOwnProperty')\n    at formatRepPar (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/queryConsulta.js:94:32)\n    at ExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/ProcSql.js:28:18)\n    at ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:22)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async ctrlExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Controler/contProcedure.js:17:29)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T16:58:35.667Z","ID_EVENTO":7320,"ID_PROCESO":6,"MSG_ERROR":"Error desconocido en Ejecucion de Procedimiento: {}.","level":"error","message":"Error ","timestamp":"2025-08-28 10:58:35"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Ha ocurrido un error inesperado.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"TypeError: Cannot read properties of undefined (reading 'hasOwnProperty')\n    at formatRepPar (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/queryConsulta.js:94:32)\n    at ExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/ProcSql.js:28:18)\n    at ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:22)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async ctrlExecProcedure (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Controler/contProcedure.js:17:29)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T16:58:36.710Z","ID_EVENTO":7321,"ID_PROCESO":6,"MSG_ERROR":"Error desconocido en Ejecucion de Procedimiento: {}.","level":"error","message":"Error ","timestamp":"2025-08-28 10:58:36"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Un registro con la información proporcionada ya existe. Por favor, verifique sus datos.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"Error\n    at Query.run (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:107:25)\n    at C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async MSSqlQueryInterface.insert (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:308:21)\n    at async model.save (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:2490:35)\n    at async Q_SURVIVOR.create (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:1362:12)\n    at async file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:38:30\n    at async obtResultado (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:123:27)\n    at async createRecord (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:37:27)\n    at async ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:16)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T19:23:14.525Z","ID_EVENTO":7322,"ID_PROCESO":2,"MSG_ERROR":"UniqueConstraintError en Ejecucion Crud/Create. Campos duplicados: 'PK_Q_SURVIVOR'. Mensaje original de DB: 'Validation error'.","level":"error","message":"Error al insertar registro Excepcion","timestamp":"2025-08-28 13:23:14"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Un registro con la información proporcionada ya existe. Por favor, verifique sus datos.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"Error\n    at Query.run (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:107:25)\n    at C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async MSSqlQueryInterface.insert (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:308:21)\n    at async model.save (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:2490:35)\n    at async Q_SURVIVOR.create (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:1362:12)\n    at async file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:38:30\n    at async obtResultado (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:123:27)\n    at async createRecord (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:37:27)\n    at async ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:16)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T19:23:14.525Z","ID_EVENTO":7322,"ID_PROCESO":2,"MSG_ERROR":"UniqueConstraintError en Ejecucion Crud/Create. Campos duplicados: 'PK_Q_SURVIVOR'. Mensaje original de DB: 'Validation error'.","level":"error","message":"Error ","timestamp":"2025-08-28 13:23:14"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Un registro con la información proporcionada ya existe. Por favor, verifique sus datos.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"Error\n    at Query.run (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:107:25)\n    at C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async MSSqlQueryInterface.insert (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:308:21)\n    at async model.save (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:2490:35)\n    at async Q_SURVIVOR.create (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:1362:12)\n    at async file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:38:30\n    at async obtResultado (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:123:27)\n    at async createRecord (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:37:27)\n    at async ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:16)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T19:25:01.462Z","ID_EVENTO":7323,"ID_PROCESO":2,"MSG_ERROR":"UniqueConstraintError en Ejecucion Crud/Create. Campos duplicados: 'PK_Q_SURVIVOR'. Mensaje original de DB: 'Validation error'.","level":"error","message":"Error al insertar registro Excepcion","timestamp":"2025-08-28 13:25:01"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Un registro con la información proporcionada ya existe. Por favor, verifique sus datos.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"Error\n    at Query.run (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:107:25)\n    at C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async MSSqlQueryInterface.insert (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:308:21)\n    at async model.save (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:2490:35)\n    at async Q_SURVIVOR.create (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:1362:12)\n    at async file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:38:30\n    at async obtResultado (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:123:27)\n    at async createRecord (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:37:27)\n    at async ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:16)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T19:25:01.462Z","ID_EVENTO":7323,"ID_PROCESO":2,"MSG_ERROR":"UniqueConstraintError en Ejecucion Crud/Create. Campos duplicados: 'PK_Q_SURVIVOR'. Mensaje original de DB: 'Validation error'.","level":"error","message":"Error ","timestamp":"2025-08-28 13:25:01"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Un registro con la información proporcionada ya existe. Por favor, verifique sus datos.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"Error\n    at Query.run (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:107:25)\n    at C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async MSSqlQueryInterface.insert (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:308:21)\n    at async model.save (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:2490:35)\n    at async Q_SURVIVOR.create (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:1362:12)\n    at async file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:38:30\n    at async obtResultado (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:123:27)\n    at async createRecord (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:37:27)\n    at async ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:16)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T19:34:20.944Z","ID_EVENTO":7324,"ID_PROCESO":2,"MSG_ERROR":"UniqueConstraintError en Ejecucion Crud/Create. Campos duplicados: 'PK_Q_SURVIVOR'. Mensaje original de DB: 'Validation error'.","level":"error","message":"Error al insertar registro Excepcion","timestamp":"2025-08-28 13:34:20"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Un registro con la información proporcionada ya existe. Por favor, verifique sus datos.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"Error\n    at Query.run (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:107:25)\n    at C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async MSSqlQueryInterface.insert (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:308:21)\n    at async model.save (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:2490:35)\n    at async Q_SURVIVOR.create (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:1362:12)\n    at async file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:38:30\n    at async obtResultado (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:123:27)\n    at async createRecord (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:37:27)\n    at async ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:16)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T19:34:20.944Z","ID_EVENTO":7324,"ID_PROCESO":2,"MSG_ERROR":"UniqueConstraintError en Ejecucion Crud/Create. Campos duplicados: 'PK_Q_SURVIVOR'. Mensaje original de DB: 'Validation error'.","level":"error","message":"Error ","timestamp":"2025-08-28 13:34:20"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Un registro con la información proporcionada ya existe. Por favor, verifique sus datos.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"Error\n    at Query.run (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:107:25)\n    at C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async MSSqlQueryInterface.insert (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:308:21)\n    at async model.save (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:2490:35)\n    at async Q_SURVIVOR.create (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:1362:12)\n    at async file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:38:30\n    at async obtResultado (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:123:27)\n    at async createRecord (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:37:27)\n    at async ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:16)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T19:40:17.387Z","ID_EVENTO":7325,"ID_PROCESO":2,"MSG_ERROR":"UniqueConstraintError en Ejecucion Crud/Create. Campos duplicados: 'PK_Q_SURVIVOR'. Mensaje original de DB: 'Validation error'.","level":"error","message":"Error al insertar registro Excepcion","timestamp":"2025-08-28 13:40:17"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Un registro con la información proporcionada ya existe. Por favor, verifique sus datos.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"Error\n    at Query.run (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:107:25)\n    at C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async MSSqlQueryInterface.insert (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:308:21)\n    at async model.save (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:2490:35)\n    at async Q_SURVIVOR.create (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:1362:12)\n    at async file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:38:30\n    at async obtResultado (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:123:27)\n    at async createRecord (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:37:27)\n    at async ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:16)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T19:40:17.387Z","ID_EVENTO":7325,"ID_PROCESO":2,"MSG_ERROR":"UniqueConstraintError en Ejecucion Crud/Create. Campos duplicados: 'PK_Q_SURVIVOR'. Mensaje original de DB: 'Validation error'.","level":"error","message":"Error ","timestamp":"2025-08-28 13:40:17"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Un registro con la información proporcionada ya existe. Por favor, verifique sus datos.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"Error\n    at Query.run (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:107:25)\n    at C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async MSSqlQueryInterface.insert (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:308:21)\n    at async model.save (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:2490:35)\n    at async Q_SURVIVOR.create (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:1362:12)\n    at async file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:38:30\n    at async obtResultado (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:123:27)\n    at async createRecord (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:37:27)\n    at async ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:16)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T19:40:25.379Z","ID_EVENTO":7326,"ID_PROCESO":2,"MSG_ERROR":"UniqueConstraintError en Ejecucion Crud/Create. Campos duplicados: 'PK_Q_SURVIVOR'. Mensaje original de DB: 'Validation error'.","level":"error","message":"Error al insertar registro Excepcion","timestamp":"2025-08-28 13:40:25"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"Un registro con la información proporcionada ya existe. Por favor, verifique sus datos.","ERROR_LINE_D":null,"ERROR_MESSAGE_D":"Error\n    at Query.run (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\mssql\\query.js:107:25)\n    at C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async MSSqlQueryInterface.insert (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:308:21)\n    at async model.save (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:2490:35)\n    at async Q_SURVIVOR.create (C:\\ZG\\Proyectos\\Desarrollo\\Poyectos Nucleo\\API\\lib_comun\\node_modules\\sequelize\\lib\\model.js:1362:12)\n    at async file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:38:30\n    at async obtResultado (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:123:27)\n    at async createRecord (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Router/Servicios/actualizaBD.js:37:27)\n    at async ejecFuncion (file:///C:/ZG/Proyectos/Desarrollo/Poyectos%20Nucleo/API/lib_comun/dist/Util/utilComun.js:14:16)","ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-28T19:40:25.379Z","ID_EVENTO":7326,"ID_PROCESO":2,"MSG_ERROR":"UniqueConstraintError en Ejecucion Crud/Create. Campos duplicados: 'PK_Q_SURVIVOR'. Mensaje original de DB: 'Validation error'.","level":"error","message":"Error ","timestamp":"2025-08-28 13:40:25"}
{"CVE_APLICACION":"NFL","CVE_TIPO_EVENTO":"E","CVE_USUARIO":"mario@quiniela.com","DESC_ERROR":"No existe valor para llave primaria  ID_QUINIELA ","ERROR_LINE_D":null,"ERROR_MESSAGE_D":null,"ERROR_NUMBER_D":null,"ERROR_PROCEDURE_D":null,"ERROR_SEVERITY_D":null,"ERROR_STATE_D":null,"F_EVENTO":"2025-08-29T19:50:00.089Z","ID_EVENTO":7327,"ID_PROCESO":7,"MSG_ERROR":"Error de tipo string en Ejecucion Crud/Bulk Update: 'No existe valor para llave primaria  ID_QUINIELA '.","level":"error","message":"Error ","timestamp":"2025-08-29 13:50:00"}
